pennylane/templates/layers/random.py

```diff
@@ -48,16 +48,18 @@ def random_layer(weights, wires, ratio_imprim, imprimitive, rotations, seed):
     while i < len(weights):
         if np.random.random() > ratio_imprim:
             # Apply a random rotation gate to a random wire
-            gate = np.random.choice(rotations)
-            wire = int(np.random.choice(wires))
-            gate(weights[i], wires=wire)
+            rnd1 = np.random.choice(range(len(rotations)))
+            gate = rotations[rnd1]
```

Comment:
Curious about this change. How come the previous way of randomly choosing a gate was changed to choosing an index at random and then indexing into `rotations`?

Reply:
You are right, I cannot remember why I did that. I changed it back to the previous more concise version.